test/reference.html

```diff
@@ -44,48 +58,72 @@
     text-align: right;
 }
 
+tr > td:nth-child(3) {
+    padding-left: .5em;
+}
 
 .col2 {
     -webkit-column-count: 2;
     -moz-column-count: 2;
     column-count: 2;
+    display: inline-block;
 }
 
 .col3 {
     -webkit-column-count: 3;
     -moz-column-count: 3;
     column-count: 3;
+    display: inline-block;
 }
 
 .col4 {
     -webkit-column-count: 4;
     -moz-column-count: 4;
     column-count: 4;
+    display: inline-block;
 }
 
 .col5 {
     -webkit-column-count: 5;
     -moz-column-count: 5;
     column-count: 5;
-}
-
-img {
-    opacity: .5;
+    display: inline-block;
```

Comment:
Yes, I was trying to get the alignment to work right. Fixed it now, don't need this anymore.